### PR TITLE
use full moment-timezone js and add test

### PR DESCRIPTION
**Pull request by kunkun-tang**
We previously used timezone-2010-to-2020 js file, which wis working. However, its name got changed when moment timezone upgraded to 0.5.13. In this change, we replace the file by the full-time-range timezone js, which will permanently stay and should not be changed in future. Also, we add some moment js tests to guarantee moment js working.

### CODE REVIEW

File: azkaban-web-server/src/web/js/azkaban/test/test.js [hunk 1]
```diff
@@ -7,8 +7,11 @@ function addClass(el, newClass) {
 var rewire = require('rewire'),
     dateJs = rewire('../util/date.js');
 
-var chai = require('chai');
-var assert = chai.assert;
+var moment = require('moment-timezone');
```
Comment (HappyRay) [inline azkaban-web-server/src/web/js/azkaban/test/test.js]: What do you think of separating independent tests into separate files?

**Comment by HappyRay** [comment]
Thanks for fixing this and adding tests!

File: azkaban-web-server/src/web/js/azkaban/test/test.js (hunk 1 above)
Comment (kunkun-tang) [inline azkaban-web-server/src/web/js/azkaban/test/test.js]: Yes. We should do it when we have large extensively tests. For now, we may keep them together.

**Comment by HappyRay** [comment]
Also is there a way to fail the build if a file being copied doesn't exist?

**Comment by kunkun-tang** [comment]
@HappyRay Unfortunately this way can not guarantee a file being always copied. I made a quick research. It looks like running nodejs and browser is quite different. In order to detect if we have an exact js file, I guess we probably need to set up browser integration test.

**Comment by HappyRay** [comment]
Is there a gradle config that will fail the gradle build process when the file is copied?

**Pull request merged**